src/auth/AuthContext.tsx

```diff
@@ -137,6 +138,7 @@ export default function AuthContextProvider({children}: PropsWithChildren<{}>) {
           return 'invalid_code';
         }
         console.warn(error);
```

Comment:
Could probably remove the console.warn, since bugsnag notify basically does the same.